### ISSUE TITLE
Double inputs when using remote play

### ISSUE THREAD

**Issue by ethan021021** [unknownskl/xbox-xcloud-client#361]
Cloud streaming works pristine! But it seems that the controller is sending double inputs when streaming my Xbox console. Any ideas on how to fix this? Thanks a bunch for your hard work :)

**Comment by ethan021021** [comment]
Never mind a reboot fixed it :)

But is there anyway to change the resolution Xbox remote play uses?

**Comment by unknownskl** [comment]
Streaming can sometimes leave the xbox in a buggy state indeed. Rebooting fixes it most of the times.
Currently this is the only supported resolution for now. Maybe in a future update it will be possible to have a higher or lower bitrate and/or resolution

**Comment by Xychen1994** [comment]
A change like this will be life-changing！But i heard Microsoft is actually capping it to 720p

**Comment by ethan021021** [comment]
> Streaming can sometimes leave the xbox in a buggy state indeed. Rebooting fixes it most of the times.
> 
> Currently this is the only supported resolution for now. Maybe in a future update it will be possible to have a higher or lower bitrate and/or resolution

Does Microsoft allow this through their API? Or is this more of a client side change?

**Comment by MisterAnderson91** [comment]
There seems to be bitrate control in the new 2.0.0-beta1 build if you want to try it from the releases page.

**Comment by Jbbrack03** [comment]
Even if Microsoft does cap it at 720p, it would be nice to see some scaling options. I play on Steam Deck and 720p leaves black bars above and below the image on the 1280x800 screen. I'm fine with 720p, but it would be nice to scale it to fit the screen.

**Comment by MisterAnderson91** [comment]
> Even if Microsoft does cap it at 720p, it would be nice to see some scaling options. I play on Steam Deck and 720p leaves black bars above and below the image on the 1280x800 screen. I'm fine with 720p, but it would be nice to scale it to fit the screen.

I'm not sure why you would want this. Xbox console streaming and cloud gaming are both locked to 16:9 so the only way to do 1280x800 would be to stretch the video vertically.

**Comment by Jbbrack03** [comment]
> > Even if Microsoft does cap it at 720p, it would be nice to see some scaling options. I play on Steam Deck and 720p leaves black bars above and below the image on the 1280x800 screen. I'm fine with 720p, but it would be nice to scale it to fit the screen.
> 
> I'm not sure why you would want this. Xbox console streaming and cloud gaming are both locked to 16:9 so the only way to do 1280x800 would be to stretch the video vertically.

You’re correct that it’s technically a stretch. But it’s not a bad one that drastically deforms the image. Quite a few games and emulators do this and it’s fine. In the instances where I’ve seen it done, it’s been worth the trade. I’d at least like to have the option. Something that can be toggled on for those that are fine with a little stretching.
